dataset/test/bins_test.cpp

```diff
@@ -71,10 +72,9 @@ TEST(DataArrayBins2dTest, concatenate_dim_2d) {
 
   EXPECT_EQ(buckets::concatenate(zy, Dim::Y), z);
   EXPECT_EQ(buckets::concatenate(zy, Dim::Z), y);
-  EXPECT_EQ(buckets::sum(
-                buckets::concatenate(buckets::concatenate(zy, Dim::Y), Dim::Z)),
-            buckets::sum(buckets::concatenate(buckets::concatenate(zy, Dim::Z),
-                                              Dim::Y)));
+  EXPECT_EQ(
+      bins_sum(buckets::concatenate(buckets::concatenate(zy, Dim::Y), Dim::Z)),
+      bins_sum(buckets::concatenate(buckets::concatenate(zy, Dim::Z), Dim::Y)));
```

Comment:
Why did you not rename the `buckets` namespace?

Reply:
Not sure (lazyness?), we have been replacing this bit by bit over the past 6 month.